components/MovieCard/index.js

```diff
@@ -4,10 +4,16 @@ import FavoriteButton from "../FavoriteButton";
 
 const Wrapper = styled.div`
   padding: 2.4rem;
-  background-color: var(--color-background-500);
+  background-color: var(--color-primary);
   border-radius: 9px;
   position: relative;
   z-index: 0;
+
+  ul {
+    font-size: 12px;
+    text-style: none;
+    margin-top: 2rem;
+  }
```

Comment:
please create a separate styled component for the ul instead of using this wrapper. This will prevent conflicting styles in the future.

Reply:
I add the changes